packages/core/src/selectors/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { ServiceContext } from "../ServiceContext";
+import { BaseSelector } from "./base";
+import { LLMMultiSelector, LLMSingleSelector } from "./llmSelectors";
+
+export const getSelectorFromContext = (
+  serviceContext: ServiceContext,
+  isMulti: boolean = false,
+): BaseSelector => {
+  let selector: BaseSelector | null = null;
+  const llm = serviceContext.llm;
+
+  if (isMulti) {
+    selector = new LLMMultiSelector({
+      llm,
+      maxOutputs: 5,
```

Comment:
In Python, it looks like this is len(choices) and not a fixed number

Reply:
on python currently maxOutputs isn't being sent

Reply:
https://github.com/run-llama/llama_index/blob/main/llama_index/selectors/utils.py#L11-L33